app/package.json

```diff
@@ -161,7 +161,10 @@
     "jest": "^29.7.0",
     "mime-types": "^2.1.35",
     "octokit": "^4.0.2",
-    "typescript": "5.5.3"
+    "typescript": "5.5.4"
+  },
+  "overrides": {
```

Comment:
Just out of curiosity, why was this needed?

Reply:
@kinvolk/eslint-config depends on typescript@4 which results in two copies of typescript in node_modules. Forcing @5 fixes that and eslint seems to work just fine